src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -105,14 +105,15 @@ class HuiGenericEntityRow extends LitElement {
                     .datetime=${stateObj.last_changed}
                   ></ha-relative-time>
                 `
-              : this.config.secondary_info === "last-triggered" &&
-                stateObj.attributes.last_triggered
-              ? html`
-                  <ha-relative-time
-                    .hass=${this.hass}
-                    .datetime=${stateObj.attributes.last_triggered}
-                  ></ha-relative-time>
-                `
+              : this.config.secondary_info === "last-triggered"
+               ? stateObj.attributes.last_triggered
+               ? html`
+                   <ha-relative-time
+                     .hass=${this.hass}
+                     .datetime=${stateObj.attributes.last_triggered}
+                   ></ha-relative-time>
+                 `
+               : this.hass.localize("ui.panel.lovelace.entities.never_triggered")
```

Comment:
this path doesn't match what you added to en.json

Reply:
ok so now I am confused. Please bare with me..
I believed you suggested this path, and so I added that to the pr of the hui-generic-entity-row.ts

following that Bram asked me to add to the translation file https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/translations/en.json

which of both should I change?

Reply:
It is the right file it is just that `ui.panel.lovelace.entities.never_triggered` isn't what you created that is: `ui.card.automation.never_triggered`

Reply:
And you should not add keys that we don't use in the code, like the other keys you added

Reply:
I see, I searched for automation and script, but should have looked for path lovelace is it? I think this is the only hat to lovelace I can find, is that what you refer to? https://github.com/home-assistant/home-assistant-polymer/blob/4e5406b27b0a14788e80859a96d2b54871954b39/src/translations/en.json#L1575 ?

if so, there is no path to entities yet? I should add this:
"entitities": {"never_triggered":"Never triggered"},

Reply:
Put it in here:
https://github.com/home-assistant/home-assistant-polymer/blob/4e5406b27b0a14788e80859a96d2b54871954b39/src/translations/en.json#L1700